packages/server/src/api/util/set-cache-control.ts

```diff
@@ -0,0 +1,61 @@
+import { Response } from 'express';
+
+export type CacheControl = {
```

Comment:
i only configured the options we're using, there are more according to the spec though